### PR TITLE
Updated readme on 10 April to demonstrate PR

### PR DESCRIPTION
**Pull request by ultimastrike**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # Services-Courses
+
 .NET 7 Minimal API for Microservices
+
+## Will Miller Microservices
+
+> 1. Some Sentence 1
+> 1. Some Sentence 2
```
Comment (ljmsgh123) [inline README.md:8]: Could you explain this sentence further?

**Review by vishipayyallore — COMMENTED** [review]
Could you please explain 2nd point

**Comment by ljmsgh123** [comment]
Please revise the sentence format.

File: README.md (hunk 1 above)
Comment (ultimastrike) [reply README.md:8]: Sure.

**Review by veeramutyala — APPROVED** [review]
Approved

**Review by jacarley — APPROVED** [review]
Looks good.

**Pull request merged**